### PR TITLE
Minor fixes in vector module

### PR DESCRIPTION
**Pull request by szymag**
Fix issue #13175,
clean up the code,
update documentation,
proper variable names.

### CODE REVIEW

**Comment by szymag** [comment]
ping @Upabjojr

File: sympy/vector/coordsysrect.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def __new__(cls, name, transformation=None, parent=None, location=None,
             The name of the new CoordSys3D instance.
 
         transformation : Lambda, Tuple, str
-            Transformation defined by transformation equations or chooesed
+            Transformation defined by transformation equations or chose
```
Comment (Upabjojr) [inline sympy/vector/coordsysrect.py]: chosen

File: sympy/vector/tests/test_coordsysrect.py [hunk 2]
```diff
@@ -463,19 +463,3 @@ def test_rotation_trans_equations():
            (-sin(q0) * c.y + cos(q0) * c.x, sin(q0) * c.x + cos(q0) * c.y, c.z)
     assert c._rotation_trans_equations(c._inverse_rotation_matrix(), c.base_scalars()) == \
            (sin(q0) * c.y + cos(q0) * c.x, -sin(q0) * c.x + cos(q0) * c.y, c.z)
-
-
-def test_translation_trans_equations():
```
Comment (Upabjojr) [inline sympy/vector/tests/test_coordsysrect.py]: why did you remove this test?

Comment (szymag) [reply sympy/vector/tests/test_coordsysrect.py]: Because I removed a method `_set_inv_trans_equations` and these tests checks only this method

File: sympy/vector/coordsysrect.py [hunk 3]
```diff
@@ -41,7 +42,7 @@ def __new__(cls, name, transformation=None, parent=None, location=None,
             The name of the new CoordSys3D instance.
 
         transformation : Lambda, Tuple, str
-            Transformation defined by transformation equations or chooesed
+            Transformation defined by transformation equations or choosen
```
Comment (Upabjojr) [inline sympy/vector/coordsysrect.py]: chosen, one o

File: sympy/vector/coordsysrect.py [hunk 4]
```diff
@@ -962,7 +997,7 @@ def create_new(self, name, transformation, variable_names=None, vector_names=Non
...
```
Comment (Upabjojr) [inline sympy/vector/coordsysrect.py]: chosen, one o

**Comment by Upabjojr** [comment]
There are some English language errors, but the code looks good.
